Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1260,38 +1278,75 @@ namespace Multiplayer
         }
     }
 
-    void MultiplayerSystemComponent::EnableAutonomousControl(NetworkEntityHandle entityHandle, AzNetworking::ConnectionId connectionId)
+    void MultiplayerSystemComponent::EnableAutonomousControl(const INetworkEntityManager::EntityList& entityList, AzNetworking::ConnectionId connectionId)
     {
-        if (!entityHandle.Exists())
+        if (entityList.empty())
         {
             AZLOG_WARN("Attempting to enable autonomous control for an invalid entity");
             return;
         }
 
-        entityHandle.GetNetBindComponent()->SetOwningConnectionId(connectionId);
-        if (connectionId == InvalidConnectionId)
+        for (NetworkEntityHandle subEntity : entityList)
```

Comment:
const ref&

Reply:
updated, the logic to enable autonomous was moved to happen before entity activation in the helper function NetworkEntityManager::CreateAutomonousPlayer